css/index.css

```diff
@@ -1,3 +1,463 @@
+  text-align: center;
+  padding: 0 75px 0 100px;
+}
+
+.cta .cta-text button {
+  background: white;
+  vertical-align: top;
+  margin-top: 6px;
+  padding: 8px 44px;
+  font-size: 17px;
+  border: 1px solid black;
+  cursor: pointer;
+}
+
+.cta .cta-text button:hover {
+  background: black;
+  color: white;
+}
+
+.cta img {
+  margin-top: 69px;
+}
+
+.main-content {
+  border-top: 1px solid black;
+  margin-top: 76px;
+  padding-top: 39px;
+}
+
+.main-content .top-content {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 10px;
+}
+
+.main-content .top-content .text-content {
+  width: 410px;
+  margin-right: 34px;
+}
+
+.main-content .top-content .text-content:last-child {
+  margin-right: 0;
+}
+
+.main-content .middle-img {
+  margin-top: 45px;
+}
+
+.main-content .bottom-content {
+  display: flex;
+  justify-content: space-between;
+  margin-top: 37px;
+  padding: 0 10px;
+}
+
+.main-content .bottom-content .text-content {
+  width: 275px;
+  margin-right: 12px;
+}
+
+.main-content .bottom-content .text-content:last-child {
+  margin-right: 0;
+}
+
+.contact {
+  margin-top: 57px;
+  border-top: 1px solid black;
+  padding: 39px 10px 0;
+}
+
+.contact p {
+  margin: 20px 0;
+}
+
+footer {
+  margin: 32px 0;
+  text-align: center;
+  font-size: 14px;
+}
+
+/* Services Page */
+
+p {
+	line-height: 1.3;
+}
+
+.services-header-img {
+	margin: 30px 0;
+}
+
+.top-info h2 {
+	font-size: 30px;
+}
+
+.info-boxes {
+	border-top: 1px solid black;
+	border-bottom: 3px solid black;
+	margin: 40px 0;
+}
+
+.service-row {
+	display: flex;
+	justify-content: space-between;
+	align-items: center;
+	margin: 40px 0;
+}
+
+.service-box {
+	border: 1px solid black;
+	background: #f3f2f3;
+	width: 420px;
+	height: 250px;
+	padding: 20px;
+}
+
+.learn-more {
+	border: 1px solid black;
+	border-radius: 4px;
+	background: white;
+	text-align: center;
+	margin: 20px 0;
+	padding: 10px 20px;
+	width: 160px;
+	font-size: 14px;
+}
+
+.btm-info {
+	display: flex;
+	justify-content: space-between;
+	margin-bottom: 70px;
+}
+
+.btm-text {
+	margin-right: 45px;
+}
+
+ul {
+	margin: 25px 0;
+	line-height: 2;
+}
+
+/* about */
+
+.about-summary h1 {
+  font-size: 30px;
+  margin: 50px 0 40px;
+  line-height: 1.5;
+}
+
+.sheep {
+  width: 880px;
+  height: auto;
+  border: 3px solid #f6f6f6;
+}
+
+.about-summary h2 {
+  font-size: 20px;
+  margin: 20px 0;
+  padding-bottom: 20px;
+  border-bottom: 4px double black;
+}
+
+.about-full {
+  display: flex;
+  flex-direction: column;
+}
+
+.about-full h2 {
+  font-size: 20px;
+}
+
+.about-full p {
+  font-size: 18px;
+}
+
+.about-text {
+  border: 1px solid grey;
+  border-radius: 2px;
+  width: 750px;
+  padding: 40px;
+  margin: 30px 0;
+}
+
+.off-center {
+  align-self: flex-end;
+  text-align: right;
+}
+
+.img-caption {
+  font-size: 18px;
+  padding-top: 15px;
+}
+
+#jacob {
+  margin-top: 20px;
+}
+
+/* Tablet View */
+
+@media (max-width: 800px) {
+
+    .container {
+        width: 740px;
```

Comment:
If you hard-code the width (especially of the container)  at a certain pixel size, it get's less responsive. Ideally, you'd want to make width 100% and then set a max-width of your wanted pixel-size